spec/requests/providers/statement_of_case_spec.rb

```diff
@@ -50,20 +50,20 @@
 
   describe 'PATCH providers/proceedings_before_the_court' do
     subject { patch providers_legal_aid_application_statement_of_case_path(legal_aid_application), params: params.merge(submit_button) }
-    let(:entered_text) { Faker::Lorem.paragraph(3) }
-
+    let(:statement) { Faker::Lorem.paragraph(3) }
+    let(:original_file) { Rack::Test::UploadedFile.new(Rails.root + 'spec/fixtures/files/lorem_ipsum.pdf', 'application/pdf') }
```

Comment:
Could we wrap this in a method defined in `rails_helper.rb` ? Something like:
```ruby
def rack_test_file(path, content_type = "text/plain", binary = false)
  path = Rails.root.join(path)
  Rack::Test::UploadedFile.new path, content_type, binary
end
```
Then here the code would be:
```ruby
let(:original_file) {  rack_test_file 'spec/fixtures/files/lorem_ipsum.pdf', 'application/pdf' }
```